web/themes/interledger/templates/page--4xx.html.twig

```diff
@@ -10,12 +10,21 @@
   <a id="main-content" tabindex="-1"></a>
   <div class="error-block">
     <h1>{{ error_code }}</h1>
-    {% if error_code == '404' %}
-    <p>{{ "Oh dear, we can't find the page you're looking for."|t }}</p>
-    {% endif %}
-    {% if error_code == '403' %}
-      <p>{{ "Unfortunately, you are not authorized to view this page. Try logging in first."|t }}</p>
+    <p>
+      {%  if error_code == "404" %}
+      {{ "Oh dear, we can't find the page you're looking for."|t }}
       {% endif %}
+      {% if error_code == "403" %}
+      {{ "Unfortunately, you are not authorized to view this page. Try logging in first."|t }}
+      {% endif %}
+    </p>
+    <section class="cta--btn">
+      <div class="content-wrapper">
+        <a class="cta--btn__link" href="/" data-umami-event="4xx page link - Home">
```

Comment:
As part of translations we need to use node references in the twig files instead of relative URLs. The reason is that then Drupal will handle navigation internally and respect language selections instead of just navigating to / will always take people to the English home page.

See and example from the footer twig file:

```
<a class="button subscribe-btn" href={{ path('entity.node.canonical', {'node': 974}) }} data-umami-event="Footer link - Subscribe">
        {{ 'Subscribe'|t }}
      </a>
 ```

To to this go to the home page and select 'Edit' and you'll see the node number in the url. Then you can use that node as the href here.